src/views/postUpdation/AllPostTypesUpdation.tsx

```diff
@@ -444,6 +479,31 @@ export const AllPostTypesUpdation: React.FC<IAllPostTypesUpdation> = ({
         disabled={isDisabled}
         post={post}
       />
+      <BorderBottom />
+      <h1>Changes List</h1>
+      <Box mt={3}>
+        <AuthorListDropdown
+          pageSizes={changesPerPage}
+          setChanges={handleNotesToShowChange}
+          selected={
+            size === 0 ? changesPerPage[changesPerPage.length - 1] : size
```

Comment:
let's go with `size || changesPerPage[changesPerPage.length - 1]`

not just shorter(code size should never be an argument per se) but also covers edge case with `undefined`(not sure whether it's allowed for this type but anyway)

Reply:
yeap im agree with you that this one way look more better